### PR TITLE
Add Life Safer

### PR DESCRIPTION
**Pull request by DivyaB1**
Hi,

Adding Life Saver - https://www.lifesafer.com/
This category looked most appropriate to me, let me know if you have any other suggestions.

Thanks,

### CODE REVIEW

File: data/brands/shop/electrical.json [hunk 1]
```diff
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^electrical$"]}
   },
   "items": [
+     {
+      "displayName": "Life Saver",
+      "locationSet": {"include": ["us","ca"]},
+      "tags": {
+        "brand": "Life Saver",
+        "brand:wikidata": "Q122257514",
+        "name": "Life Saver",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/electrical.json:13]:
```suggestion
      "displayName": "LifeSafer",
      "locationSet": {"include": ["us","ca"]},
      "tags": {
        "brand": "LifeSafer",
        "brand:wikidata": "Q122257514",
        "name": "LifeSafer",
```
The brand is LifeSafer, not Life Saver

File: data/brands/shop/electrical.json [hunk 2]
```diff
@@ -4,6 +4,16 @@
...
+        "shop": "electrical"
```
Comment (AntiCompositeNumber) [inline data/brands/shop/electrical.json:14]: Hm. I'm not sure `shop=electrical` is the best tag here. `shop=car_repair` with some value of `service=` would fit better to me, but I'm not sure. Of already-mapped locations, one is tagged `shop=car_parts` and the rest have no clear specific tag.

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]

File: data/brands/shop/electrical.json (hunk 1 above)
Comment (DivyaB1) [reply data/brands/shop/electrical.json:13]: Yup, I agree. Added the space by mistake.

**Comment by DivyaB1** [comment]
Looks like the website does show stores that sell and install LifeSaver as opposed to having standalone locations.
Closing the PR.

**Pull request closed without merging**